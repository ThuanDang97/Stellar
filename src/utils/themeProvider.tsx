import { ThemeProvider } from 'styled-components'
import { render } from '@testing-library/react'
import { theme } from '@themes/index'

export const renderWithTheme = (component: React.ReactNode) => {
  return render(<ThemeProvider theme={theme}>{component}</ThemeProvider>)
}
