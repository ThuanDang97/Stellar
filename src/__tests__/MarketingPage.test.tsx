import React from 'react'

// Utils
import { renderWithTheme } from '@utils/themeProvider'

// Page
import Marketing from '@pages/marketing'

describe('Marketing Page', () => {
  it('Should be renders Marketing page ', () => {
    const { asFragment } = renderWithTheme(<Marketing />)

    expect(asFragment()).toMatchSnapshot()
  })
})
