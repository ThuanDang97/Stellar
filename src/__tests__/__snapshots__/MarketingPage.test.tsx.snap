// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Marketing Page Should be renders Marketing page  1`] = `
<DocumentFragment>
  <div
    class="ViewStyled-sc-rqe5sa-0 ViewStyled__WrapperContent-sc-rqe5sa-6 kBewKp gVWyht"
    display="flex"
  >
    <div
      class="ViewStyled-sc-rqe5sa-0 knVWgS"
      display="flex"
    >
      <div
        class="ViewStyled-sc-rqe5sa-0 knVWgS"
        display="flex"
      >
        <span
          class="TextStyled-sc-9xia41-0 exqdNj"
          color="#203F59"
          data-testid="stepper-value"
          font-weight="400"
        >
          Sign Up
        </span>
        <span
          class="TextStyled-sc-9xia41-0 StepperStyled__ArrowStyled-sc-1rhowde-0 gtHdGO jbeCTy"
        />
      </div>
      <div
        class="ViewStyled-sc-rqe5sa-0 knVWgS"
        display="flex"
      >
        <span
          class="TextStyled-sc-9xia41-0 leTdwL"
          color="#C0C0C0"
          data-testid="stepper-value"
          font-weight="400"
        >
          Set Goals
        </span>
        <span
          class="TextStyled-sc-9xia41-0 StepperStyled__ArrowStyled-sc-1rhowde-0 gtHdGO jbeCTy"
        />
      </div>
      <div
        class="ViewStyled-sc-rqe5sa-0 knVWgS"
        display="flex"
      >
        <span
          class="TextStyled-sc-9xia41-0 leTdwL"
          color="#C0C0C0"
          data-testid="stepper-value"
          font-weight="400"
        >
          Connect Bank
        </span>
      </div>
    </div>
    <h1
      class="TextStyled-sc-9xia41-0 TitleStyled-sc-t58pjn-0 eZwSKl feyBag"
      color="#242A30"
      font-family="AdobeCleanExtraBold"
      font-size="36"
    >
      How did you hear about Stellar?
    </h1>
    <span
      class="TextStyled-sc-9xia41-0 dVKvcV"
      color="#4C5156"
      font-size="16"
    >
      Select all that apply.
    </span>
    <div
      class="ViewStyled-sc-rqe5sa-0 ViewStyled__BoxShadowStyled-sc-rqe5sa-2 kRzxHL dbZetz"
      display="flex"
    >
      <div
        class="ViewStyled-sc-rqe5sa-0 fEXkeO"
        display="flex"
      >
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Podcast
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Referral
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Clubhouse
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Pinterest
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              YouTube
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Facebook
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Online Ad
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Wed Search
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Instagram
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              TikTok
            </span>
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 gWlwSj"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 hBwktS"
            cursor="pointer"
            data-testid="label-box"
          >
            <span
              class="TextStyled-sc-9xia41-0 dItzST"
              color="#242A30"
              font-size="14"
            >
              Other
            </span>
          </div>
        </div>
      </div>
      <button
        class="ButtonStyled-sc-1ys9v52-0 xbTIZ"
        data-testid="button"
        disabled=""
      >
        Continue
      </button>
    </div>
  </div>
</DocumentFragment>
`;
